src/olympia/versions/models.py

```diff
@@ -549,7 +549,7 @@ def reset_nomination_time(self, nomination=None):
 
     @property
     def is_listed(self):
-        return self.addon.is_listed
+        return self.channel == amo.RELEASE_CHANNEL_LISTED
 
```

Comment:
`link_if_listed_else_text` macro is still using it :( I'll revert that change and clean that up in a separate PR.